assets/ontologies/CPV/latest

```diff
@@ -1 +1 @@
-v0.7
\ No newline at end of file
+v0.7
```

Comment:
Something happened here that broke the symlinks (also the other ones changed in this PR), the GitHub UI isn't showing the whitespaces.

Besides, if this is a tools repo it shouldn't have ontologies in it.

Reply:
I haven't changed anything here. This file already existed before.